browser/lib/i18n.js

```diff
@@ -1,11 +1,17 @@
 const path = require('path')
 const { remote } = require('electron')
 const { app } = remote
+const { languages } = require('./Languages.js')
+
+let locales = languages.reduce(function (localeList, locale) {
```

Comment:
`locales` is never reassigned. Please use `const` instead of `let`.

Reply:
And this duplicates with [this lines](https://github.com/ZeroX-DG/Boostnote/blob/65d8d7282f8f8fad5ff23a31b432a5a97d222a6f/browser/main/Main.js#L156-L159). 
Please export `locales` from this file and import `locales` from this file in `browser/main/Main.js`.